File: code/c/control-flow/simple-case.c
/* Program: simple-case.c */
#include <stdio.h>

int main()
{
    char ch;
    printf("Enter a character: ");
    scanf("%c", &ch);
    
    switch(ch)
    {
        case 'a': 
        case 'b': printf("A or B\n");
                  break;
        case 'c': printf("C ");
        case 'd': printf("and D\n");
        default:  printf("Something else...\n");
    }
    return 0;
}

File: code/pascal/control-flow/SimpleCase.pas
program SimpleCase;

procedure Main();
var
    ch: Char;
begin
    Write('Enter a character: ');
    ReadLn(ch);
    
    case ch of
      'a', 'b':           WriteLn('a or b');
      'c', 'e':           WriteLn('c or e');
      'd':                WriteLn('d');
      'f'..'z', 'F'..'Z': WriteLn('f to z or F to Z')
      else                WriteLn('Something else...');
    end;
end;

begin
  Main();
end.